webapp/src/components/CollectionPage/CollectionPage.tsx

```diff
@@ -41,7 +41,7 @@ const CollectionPage = (props: Props) => {
   return (
     <div>
       <Navbar isFullscreen />
-      <Navigation activeTab={NavigationTab.MY_STORE} />
+      <Navigation activeTab={NavigationTab.COLLECTIBLES} />
```

Comment:
Hi @nishit-g! The collection detail can be accessed from different parts of the app (like from an item detail, or from the collections section of My Store). This change in the PR makes it consistent when you navigate from an item detail to a collection detail (in both cases the Collectibles tab is active), but now it is inconsistent when you navigate from your collections' list in My Store (it switches back and forth between My Store and Collectibles).

I think the best solution is to turn this tab off in the collection detail rather than keeping Collectibles or My Store active.

```suggestion
      <Navigation />
```